api/v3/licensemaster_types.go

```diff
@@ -28,6 +30,12 @@ import (
 // Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 // see also https://book.kubebuilder.io/reference/markers/crd.html
 
+const (
+	// LicenseMasterPausedAnnotation is the annotation that pauses the reconciliation (triggers
```

Comment:
Same as for [comment](https://github.com/splunk/splunk-operator/pull/612/files#r773377479)

Reply:
fixed